src/strongbox/ui/jfx.clj

```diff
@@ -1849,53 +1898,69 @@
 
 (defn search-addons-search-field
   [{:keys [fx/context]}]
-  (let [search-state (fx/sub-val context utils/just-in [:app-state :search [:term :filter-by :page :results-per-page]])
+  (let [search-state (fx/sub-val context get-in [:app-state, :search])
+        ;;known-host-list (fx/sub-val context get-in [:app-state, :db-stats :known-host-list])
         known-host-list (core/get-state :db-stats :known-host-list)
```

Comment:
erm - this should change the available hosts in the host drop down, it's not. fix/revert

Reply:
actually it is, but I have starred addons. hrm.